core/lib/utils/html/html_utils.dart

```diff
@@ -153,60 +153,118 @@ class HtmlUtils {
+    }
+
+    #pdf-viewer {
+      flex: 1; /* Allow viewer to fill remaining space */
+      border: 1px solid #ddd;
+      margin-left: auto;
+      margin-right: auto;
+      padding-top: 53px;
+      border: none;
+    }
+
+    #app-bar {
+      position: fixed; /* Fix app bar to top */
+      top: 0;
+      left: 0;
+      right: 0; /* Stretch across entire viewport */
+      display: flex;
+      justify-content: space-between;
+      padding: 8px 8px;
+      background-color: #323639;
+      z-index: 100; /* Ensure buttons stay on top */
+      align-items: center;
+    }
+
+    .btn {
+      padding: 5px 8px;
+      border: none;
+      border-radius: 50%;
+      cursor: pointer;
+      margin-left: 5px;
+      background-color: transparent;
+    }
+    
+    .btn:hover {
+      background-color: #555;
+    }
+
+    #file-info {
+      width: 30%;
+      display: flex;
+      align-items: center;
+      padding: 5px 10px;
+    }
+
+    #file-name {
+      font-size: .87rem;
+      font-weight: 500;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      display: -webkit-box;
+      -webkit-line-clamp: 2;
+      line-clamp: 2;
+      -webkit-box-orient: vertical;
+      color: #fff;
+    }
+    
+    #file-size {
+      font-size: .87rem;
+      font-weight: 200;
+      color: #ddd;
+    }
+  ''';
+
   static String chromePdfViewer(Uint8List bytes, String fileName) {
     return '''
       <!DOCTYPE html>
       <html lang="en">
         <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta charset="utf-8" />
         <title>$fileName</title>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/pdf.js/4.0.379/pdf.min.mjs" type="module"></script>
         <style>
-          body {
-            background-color: black;
-          }
-
-          #pdf-container {
-            $_pdfContainerStyle
-          }
-
-          #pdf-viewer {
-            $_pdfViewerStyle
-          }
-
-          #app-bar {
-            $_pdfAppBarStyle
-          }
-
-          #download-btn {
-            $_pdfDownloadButtonStyle
-          }
-
-          #file-info {
-            $_pdfFileInfoStyle
-          }
-
-          #file-name {
-            $_pdfFileNameStyle
-          }
+          $_pdfViewerCssStyle
         </style>
         </head>
         <body>
           <div id="pdf-container">
-            $_pdfAppbarElement
+            ${_pdfAppbarElement(isSupportedPrint: true)}
             <div id="pdf-viewer"></div>
           </div>
 
           <script type="module">
             function renderPage(pdfDoc, pageNumber, canvas) {
               pdfDoc.getPage(pageNumber).then(page => {
-                const viewport = page.getViewport({ scale: 1 });
-                canvas.height = viewport.height;
-                canvas.width = viewport.width;
-
-                const context = canvas.getContext('2d');
-                const renderContext = {
+                var scale = 1.5;
```

Comment:
why `1.5`?

Reply:
This is the default value to fix PDF viewer render blurry on Chrome/Opera/Edge error